common/lib/helpers.py

```diff
@@ -881,3 +898,9 @@ def folder_size(path='.'):
         elif entry.is_dir():
             total += folder_size(entry.path)
     return total
+
+def hash_values(string: str) -> str:
```

Comment:
Since we are using several types of hashes in 4CAT (MD5, SHA256 and Blake2b) please indicate in the function name that this is md5. And perhaps `value` instead of `values`?